README.md

```diff
@@ -32,12 +32,19 @@ $ go install github.com/osbuild/image-builder-cli/cmd/image-builder@main
 
 We plan to provide rpm packages in fedora as well.
 
+## Compilation
+
+You can as well compile the application from local source with
```

Comment:
Its probably good to add comething like `You can compile the application in cmd/image-builder with the normal go commandd` (or spell out the commands). Just to avoid the impression that there is anything special needed that requires a makefile (those are very usual IME in go projects)

Reply:
I tend to document copy'n'paste ready commands and with the make command we can adapt it and do other related things like create a `bin` directory (uh which should be added to `.gitignore`, I just saw) and also test it with all the tags needed to build (with github actions)

Reply:
Sure, but please also mention something like "You can compile the application in cmd/image-builder with the normal go command" just to avoid the impression that there is anything special needed that requires a makefile (those are very usual IME in go projects).